roles/nginx/tasks/mviewer.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Checkout mviewer
+  git:
+    repo: "{{ mviewer.gitrepo }}"
+    version: "{{ mviewer.gitversion }}"
+    dest: /var/www/mviewer
+  when: mviewer.enabled
+
+- name: Create a geOrchestra group
+  become: yes
+  group: 
+    name: georchestra
+    state: present
+
+- name: Create a user for Mviewer and MviewerStudio
+  become: yes
+  user:
+    name: mviewer
+    state: present
+
+- name: Change ownership
+  file:
+    path: /var/www/mviewer
+    owner: mviewer
+    group: georchestra
+    state: directory
+    recurse: yes
```

Comment:
no need for that task, the user running the code (mviewer) shouldnt need to have write access to its code

mviewerstudio should store its data in `/srv/data/mviewer` which should be the only dir he needs write access